data/accounts/sv_SE/bas_2019_k1.gnucash-xea

```diff
@@ -0,0 +1,4040 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<gnc-account-example
+	xmlns:gnc-act="uri:gnucash" xmlns:act="uri:act" xmlns:gnc="uri:gnc" xmlns:cmdty="uri:cmdty" xmlns:slot="uri:slot">
+    <gnc-act:title>
+      Kontoplan BAS 2019 K1
+    </gnc-act:title>
+    <gnc-act:short-description>
+      Kontoplan BAS 2019 K1
+    </gnc-act:short-description>
+    <gnc-act:long-description>
+      BAS 2019 - Förenklat årsbokslut
+
+För mer information om BAS kontoplaner se http://www.bas.se/.
+    </gnc-act:long-description>
+<gnc-act:exclude-from-select-all>1</gnc-act:exclude-from-select-all>
+<gnc:account version="2.0.0">
+  <act:name>Root Account</act:name>
+  <act:id type="guid">5df07084fb48f599732df308a71d8246</act:id>
```

Comment:
Also, when I replace "type"="guid" with "type"="new", I should delete the actual GUID of the element, right?

And use  <act:id type="new" />   ?

Reply:
> Also, when I replace "type"="guid" with "type"="new", I should delete the actual GUID of the element, right?
> 
> And use <act:id type="new" /> ?

That is a bad idea because the current guids are used in the child accaounts:
`<act:parent type="guid">5df07084fb48f599732df308a71d8246</act:parent>`
If you remove them, the relation gets lost.
Just set the attrib type="new" and the program will do the replacement for you.

Reply:
Yes, I realised that this would have created dangling references from the parent elements, but only after having almost finished (incorrectly) editing one of the files. 

Should I give it a try and make it a little more explicit it on the wiki page?

Also, if the XSL transform in gnc2xea.xsl were to take this into account, the step of changing the attribute value wouldn't be needed right? Is there a reason (other than lack of time) that gnc2xea.xsl hasn't been updated to change the attribute value?

Reply:
> Yes, I realised that this would have created dangling references from the parent elements, but only after having almost finished (incorrectly) editing one of the files.

Sorry for my late answer.
> Should I give it a try and make it a little more explicit it on the wiki page?

Yes.
> Also, if the XSL transform in gnc2xea.xsl were to take this into account, the step of changing the attribute value wouldn't be needed right? Is there a reason (other than lack of time) that gnc2xea.xsl hasn't been updated to change the attribute value?

The xsl was simply forgotten for a long time.